src/config/_constants.py

```diff
@@ -0,0 +1,13 @@
+"""
+_constants -   # FIXME teplates
+
+
+
+Copyright (c) 2019 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+_PLUGIN_NAME = "mecha_config"
```

Comment:
[Question] Is there a reason this is labeled a private variable?
It seems to be the exact kind of thing this module *wants* to export.

Reply:
no, external plugins shouldn't need to access the plugins name, further it is intended to only be used internally by the package's modules.

Reply:
Still feels odd to have an entire file dedicated to one constant value, and importing private vars is always a bit of a code smell imo.

Reply:
this name is required internally by other modules, to prevent duplicity i needed to place it somewhere the other internals could access. The alternative here is to duplicate the hardcoded string across multiple modules.

I don't see a reason in which something external to this plugin system needs to access this name.

Reply:
I agree with Eearsy here, but I can't offer a better alternative.

Reply:
If you guys can come up with a better alternative, im all ears.